docs/src/modeling/bausteine/integrationen/index.md

```diff
@@ -106,4 +109,133 @@ Für die Interaktion mit dem S3-Dienst stehen verschiedene Templates zur Verfüg
+| Title                         | Name des Vorgangs                                                     | Vorgangsname            |
+| Betreff                       | Betreff des Vorgangs                                                  | Vorgangsbetreff         |
+| User                          | Benutzername des Benutzers, über den der Vorgang angelegt werden soll | max.mustermann          |
+| Sachakte (Coo)                | Objekt-ID der Sachakte, in der der Vorgang erstellt werden soll       | COO.1234.5678.9.1234567 |
+| Out: Procedure (Coo)          | Objekt-ID des angelegten Vorgangs                                     | COO.9876.5432.1.9876543 |
+
+### Dokument anlegen
+
+Um ein Dokument anzulegen, wird das Element Template `DMS: Dokument anlegen` verwendet.
+
+**Properties**
+
+| Property                      | Beschreibung                                                           | Beispiel                             |
+|-------------------------------|------------------------------------------------------------------------|--------------------------------------|
+| Dms System (Integration Name) | Dropdown-Auswahl zwischen MUCS und ALW DMS                             | mucs bzw. alw                        |
+| Vorgang (Coo)                 | Objekt-ID des Vorgangs, in dem das Dokument erstellt werden soll       | COO.1234.5678.9.1234567              |
+| Title                         | Name des Dokuments                                                     | Dokumentname                         |
+| Datum                         | Eingang- bzw. Ausgangsdatum des Dokuments                              | 2024-01-31                           |
+| User                          | Benutzername des Benutzers, über den das Dokument angelegt werden soll | max.mustermann                       |
+| Typ (Ein-/Ausgehend/Intern)   | Dropdown-Auswahl zwischen Eingangs-, Ausgangs- und internem Dokument   | Eingehend bzw. Ausgehend bzw. Intern |
+| Pfad(e) im S3                 | Pfad(e) zu Dateien oder Ordnern im S3 mit einem Komma getrennt         | ordnername/,odner/filename.pdf       |
+| Out: Document (Coo)           | Objekt-ID des Dokuments                                                | COO.9876.5432.1.9876543              |
+
+### Dokument updaten
+
+Um ein Dokument upzudaten, wird das Element Template `DMS: Dokument updaten` verwendet.
+
+**Properties**
+
+| Property                      | Beschreibung                                                            | Beispiel                             |
+|-------------------------------|-------------------------------------------------------------------------|--------------------------------------|
+| Dms System (Integration Name) | Dropdown-Auswahl zwischen MUCS und ALW DMS                              | mucs bzw. alw                        |
+| Dokument (Coo)                | Objekt-ID des Dokuments, das upgedatet werden soll                      | COO.1234.5678.9.1234567              |
```

Comment:
In der CoCreation heißt das Feld `Dokument-COO`.
Bis es angeglichen ist, sollte die Bezeichnung in der Doku die gleiche wie in er App sein.